### PR TITLE
translations: add missing speech mark and make po

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: locale/de_DE/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -4150,10 +4150,10 @@ msgid ""
 "and follow the invitation link again. If you decline the invitation the link "
 "is not valid anymore."
 msgstr ""
-"Wenn Sie akzeptieren, treten Sie dem Projekt mit Ihrem Nutzernamen "
-"\"%(user)s\" bei. Wenn Sie dem Projekt mit einem anderen Profil "
-"beitreten möchten, melden Sie sich ab und folgen Sie dem Einladungslink
```
Comment (phillimorland) [inline locale/de_DE/LC_MESSAGES/django.po:4155]: there was a missing `"` here which may been why it wasn't picking up a4 translations?`anyway they all seem tot be there now

**Review by sabinammm — DISMISSED** [review]
Yes, I also did a makemessages NOW that brought in the polygon translations from a4. I don't understand why these didn't show up before, as they were merged in a4 5 days ago and mB updated to this a4 then...
Thanks for fixing it :)

**Comment by sabinammm** [comment]
@phillimorland The failing test regards the BudgetingProposalListItem.jsx. Is this from a change in a previous merged PR?

**Review by sabinammm — APPROVED** [review]
Yes!! Working:)

**Pull request merged**